libs/ui/src/hooks/smartcard_auth/use_dipped_smartcard_auth.test.ts

```diff
@@ -419,4 +470,94 @@ describe('useDippedSmartcardAuth', () => {
       })
     );
   });
+
+  it('returns logged_out auth when machine is not configured and admin is not allowed to access unconfigured machines', async () => {
+    mockOf(areVvsg2AuthFlowsEnabled).mockImplementation(() => true);
+    const cardApi = new MemoryCard();
+    const logger = fakeLogger();
+
+    cardApi.insertCard(makeAdminCard(electionHash));
+    const { result, waitForNextUpdate } = renderHook(() =>
+      useDippedSmartcardAuth({
+        cardApi,
+        logger,
+        scope: { electionDefinition: undefined },
+      })
+    );
+    jest.advanceTimersByTime(CARD_POLLING_INTERVAL);
```

Comment:
Do we want to keep doing this?

Reply:
Yes good q - I know I still need to close the loop on https://github.com/votingworks/vxsuite/pull/2201#discussion_r930445881

I'd initially been thinking I'd avoid these `jest.advanceTimersByTime(CARD_POLLING_INTERVAL)` calls in top-level UI tests, but I see no reason why we shouldn't also avoid them in more internal tests like this - will update!